scipy/stats/_relative_risk.py

```diff
@@ -0,0 +1,197 @@
+        The confidence interal is computed using the Katz method
+        ([1]_, [2]_).
+
+        Parameters
+        ----------
+        confidence_level : float, optional
+            The confidence level to use for the confidence interval.
+            Default is 0.95.
+
+        Returns
+        -------
+        ci : namedtuple
+            The return value is a namedtuple with named fields
+            ``low`` and ``high``.
+
+        References
+        ----------
+        .. [1] Hardeo Sahai and Anwer Khurshid, Statistics in Epidemiology,
+               CRC Press LLC, Boca Raton, FL, USA (1996).
+        .. [2] D. Katz, J. Baptista, S. P. Azen and M. C. Pike, "Obtaining
+               confidence intervals for the risk ratio in cohort studies",
+               Biometrics, 34, 469-474 (1978).
+
+        Examples
+        --------
+        >>> from scipy.stats import relative_risk
+        >>> result = relative_risk(exposed_cases=10, exposed_total=75,
+        ...                        control_cases=12, control_total=225)
+        >>> result.relative_risk
+        2.5
+        >>> result.confidence_interval()
+        ConfidenceInterval(low=1.1261564003469628, high=5.549850800541033)
+        """
+        if not 0 <= confidence_level <= 1:
+            raise ValueError('confidence_level must be in the interval '
+                             '[0, 1].')
+
+        alpha = 1 - confidence_level
+        z = ndtri(1 - alpha/2)
+        rr = self.relative_risk
+        # Estimate of the variance of log(rr) is
+        # var(log(rr)) = 1/exposed_cases - 1/exposed_total +
+        #                1/control_cases - 1/control_total
+        # and the standard error is the square root of that.
+        se = np.sqrt(1/self.exposed_cases - 1/self.exposed_total +
+                     1/self.control_cases - 1/self.control_total)
+        delta = z*se
+        katz_lo = rr*np.exp(-delta)
+        katz_hi = rr*np.exp(delta)
+        return ConfidenceInterval(low=katz_lo, high=katz_hi)
+
+
+def relative_risk(exposed_cases, exposed_total, control_cases, control_total):
+    """
+    Compute the relative risk (also known as the risk ratio).
+
+    This function computes the relative risk associated with a 2x2
+    contingency table.  Instead of accepting a table as an argument, the
+    individual numbers that are used to compute the relative risk are
+    given as separate parameters.  This is to avoid the ambiguity of which
+    row or column of the contingency table corresponds to the "exposed"
+    cases and which corresponds to the "control" cases.  Unlike, say, the
+    odds ratio, the relative risk is not invariant under an interchange of
+    the rows or columns.
+
+    Parameters
+    ----------
+    exposed_cases : int
+        The number of "cases" (i.e. occurrence of disease or other event
+        of interest) among the sample of "exposed" individuals.
+    exposed_total : int
+        The total number of "exposed" individuals in the sample.
```

Comment:
not sure if this is the best spot in docstring but maybe mention the `>0` s?

Reply:
I updated the docstring to note the constraints on those inputs.